backend/src/v2/metadata/client.go

```diff
@@ -846,14 +855,25 @@ func (c *Client) GetExecutionsInDAG(ctx context.Context, dag *DAG, pipeline *Pip
 		glog.V(4).Infof("execs: %v", execs)
 		for _, e := range execs {
 			execution := &Execution{execution: e}
-			taskName := execution.TaskName()
+			taskName := UpdateTaskNameWithDagID(execution.TaskName(), parentDAGID)
+			glog.V(4).Infof("taskName: %s", taskName)
```

Comment:
Do you think it might be helpful to also log the `taskName` before the modification? e.g. `taskName before DAG injection:` and `taskName after DAG injection:`.